### DISCUSSION TITLE
Welcome to EDPN Discussions!

### DISCUSSION THREAD

**Discussion by llamaair** [eddb2-0/EDDB-2.0-frontend#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by madmax-codewarrior-0** [comment]
Hey everyone. I saw the news about EDDB and its developer/maintainer not being able to continue supporting the resource anymore, and like many others I was shocked and semi-devastated. EDDB was the tool I used to make billions of credits in game (thank you trade-loops and mining (well... and the Smeaton Orbital runs before distance-to-star was nerf for passenger missions)). It became a core tool, and why I **_always_** had a second monitor open to ED tools like EDDB, EDSM, INARA, etc.

I hate to see it go, and so that's why I'm here to help out. Also like the original EDDB developer/maintainer, I don't get to play the game as much as I wish I could anymore, but I'm on the idea that I'd like to be able to give back to the community, since the community is really what keeps the game and the tools alive. 

I'm hoping my experience with my current and previous positions for work will allow me to aid in the maintenance and design of the platform that the new EDDB tool sits on.
